src/Directory.Build.props

```diff
@@ -28,9 +28,9 @@
     <IsTargetFrameworkNet8OrGreater Condition="$([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)')) >= 8">true</IsTargetFrameworkNet8OrGreater>
     <AppBuildTFMs>netcoreapp3.1;net8.0</AppBuildTFMs>
     <AppBuildTFMs Condition="'$(CIBuildReason)' == 'CI'">netcoreapp3.1;net8.0</AppBuildTFMs>
-    <LibBuildTFMs>netstandard2.0;net8.0</LibBuildTFMs>
+    <LibBuildTFMs>netstandard2.0;net8.0;net9.0</LibBuildTFMs>
     <LibBuildTFMs Condition="'$(CIBuildReason)' == 'CI'">netstandard2.0;net8.0</LibBuildTFMs>
```

Comment:
Don't you also want to enable it for CI?

Reply:
Ah, yup! I think this was from before the CI images had .NET 9 😅